modules/countdown.js

```diff
@@ -3,46 +3,78 @@ require('moment-countdown')
 
 let timeout = null
 
-const ARTICLE_50 = '2020-01-31T00:00:00+01:00'
+function isPast (date) {
+  const now = moment()
+  const expiryDate = moment(date)
+  return expiryDate.isBefore(now)
+}
+
+function nextDate (milestones) {
+  for (const milestone in milestones) {
+    if (!isPast(milestone.date)) return milestone
+  }
+  return { name: null, date: null }
+}
+
+function nextBrexitDate () {
+  const milestones = [
+    { name: 'Article 50', date: '2020-01-31T00:00:00+01:00' },
+    { name: 'The transition period', date: '2019-12-31T00:00:00+01:00' }
+  ]
+  return nextDate(milestones)
+}
 
 function autoCount (bot, lastTick) {
-  const thisTick = moment(ARTICLE_50).countdown().toString().split(/, | and /)[0]
+  const { name, date } = nextBrexitDate
+  if (!name) return
```

Comment:
Do we need to clear up `timeout` here? I don't know much about how `setTimeout` actually works

Reply:
I don't think so. As long as we don't call `setTimeout` again we should be fine.